src/element.ts

```diff
@@ -521,7 +544,11 @@ export class Element {
   renderText(ctx: RenderContext, xPos: number, yPos: number): void {
     ctx.save();
     ctx.setFont(this.textFont);
-    ctx.fillText(this.text, xPos + this.xShift, yPos + this.yShift);
+    ctx.fillText(this.text, xPos + this.x + this.xShift, yPos + this.y + this.yShift);
+    this.children.forEach((child) => {
```

Comment:
Can we also clean up the grammar/formatting for the comment on lines 78-79 of this file? (e.g., "they are responsible for").

```js
  // all Element objects keep a list of children that they are responsible and which
  // inherit the style of their parents.
  protected children: Element[] = [];
```

Maybe something like:

```js
  // Element objects keep a list of children that they are responsible for.
  // Children inherit the style from their parents (see: setGroupStyle(s)).
  protected children: Element[] = [];
```

Reply:
done